### DISCUSSION TITLE
Will it also work on X10 ismr?

### DISCUSSION THREAD

**Discussion by ok5kwt** [EdgeTX/edgetx#1345] [Q&A] [answered]
Will the Edge also work on the Horus X10 with the ISMR Access module?
Will the menu also be in Czech?
Thank you

**Comment by pfeerick** [comment, marked as answer]
- ISMR should be an option in the Radio/SYS menu at the bottom of the hardware tab for the internal module
- Menu will be in English until 2.6.0 is released, at which point all the languages will be built/made available

**Comment by daveelsey** [comment]
My X12S wouldn't bind in 2.4 ACCESS but this fix worked. HOWEVER, I can't get the R9M -2019 external to bind. Any ideas? The only option I see and used is "R9M" and the LED on it does light. Thx
